lib/ahoy/stores/mongoid_store.rb

```diff
@@ -15,7 +15,7 @@ def track_visit(options, &block)
         yield(visit) if block_given?
 
         visit.upsert
-        geocode(visit)
+        geocode(visit.id.data)
```

Comment:
Since the BSON::Binary class is a concern of mongoid, shouldn't this line change to something like
```ruby
geocode(Visit.find( BSON::Binary.new( visit.id.data ,:uuid) ))
```
so that the GeocodeJob could remain the same and unconcerned with BSON::Binary?